crates/biome_js_analyze/src/lib.rs

```diff
@@ -198,18 +198,18 @@ mod tests {
 
     use super::*;
 
-    #[ignore]
+    // #[ignore]
```

Comment:
This should remain ignored.

```suggestion
    #[ignore]
```